client/signup/steps/site-type/index.jsx

```diff
@@ -11,7 +11,7 @@ import SiteTypeForm from './form';
 
 const siteTypeToFlowname = {
 	import: 'import-onboarding',
-	'online-store': 'ecommerce-onboarding',
+	'online-store': 'ecommerce',
```

Comment:
It seems that if the site type is `online-store`, the user has a chance to enter the `ecommerce-onboarding` flow. So I replace it with `ecommerce` to avoid any potential problem.